### DISCUSSION TITLE
iTwin Exporter for Datasmith crashing on export

### DISCUSSION THREAD

**Discussion by WillProicere** [iTwin/itwinjs-core#5340] [Q&A]
A few days ago I was trying to export a SYNCHRO model and the app was crashing so I decided to export it in sections which seemed to work.

Today I can't export a single object without it crashing, so I reinstalled it and the results the same, I can export with all objects hidden but with anything not hidden it gets to 3/4 of the way done and then closes.

It creates an Assets folder with some jpegs but no .udatasmith file.

Tried it with a different model and it managed to export it fine and did it quickly so it seems like it's an issue with the model

**Comment by arnab-git-hub** [comment]
@MichaelBelousov  Can you please look into this issue?

**Reply by MichaelBelousov** [reply]
@WillProicere could you reach out to me via [email](mailto:mike.belousov@bentley.com) to discuss how I could test with your dataset?
